pybliometrics/scopus/utils/get_content.py

```diff
@@ -163,17 +173,22 @@ def get_folder(api, view):
     from configparser import NoOptionError
     from pathlib import Path
 
-    from pybliometrics.scopus.utils import CONFIG_FILE, DEFAULT_PATHS
-    from pybliometrics.scopus.utils.create_config import create_config
+    from pybliometrics.scopus.utils import DEFAULT_PATHS
+
+    config = get_config()
+    config_path = get_config_path()
 
     if not config.has_section('Directories'):
-        create_config()
+        raise FileNotFoundError(
+            'No configuration file found, please create one by initialize the scopus library with'
+            'init()'
+        )
```

Comment:
It's not the scopus library but the pybliometrics library. And please make the message a separate object.